docs/Upgrade-Project-From-V1.md

```diff
@@ -1,20 +1,20 @@
-# Upgrade Guide For ask-cli beta (askx)
- 
-> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI Beta (askx)**
- 
-Skills created with CLI 1.x will need to update their project structure in order to deploy with the CLI Beta (askx). To help projects upgrade to the new format, we have provided an `askx util upgrade-project` command that will attempt to perform the necessary changes.  
- 
+# Upgrade Guide For ask-cli beta (ask)
```

Comment:
I think those are not beta anymore. All the beta word in this page need to be removed